dist/main.d.ts

```diff
@@ -0,0 +1,16 @@
+import './index.d.ts';
```

Comment:
Historically we've been just adding things like this to the top of `index.d.ts`:
https://github.com/gpuweb/types/blob/8faeb6d6df4d6f577a34a65b3bf768b8bd2714cd/dist/index.d.ts#L5-L49
It's probably fine to have multiple files, but we should have all of the additions in one place.

Reply:
moved what I could.